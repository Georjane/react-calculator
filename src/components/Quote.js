import React from 'react';
import '../styles/Quote.css';

const Quote = () => (
  <div className="quote">
    <h2>
      <blockquote>
        Mathematics is not about numbers, equations, computations,
        or algorithms: it is about understanding. â€“William Paul Thurston
      </blockquote>
    </h2>
  </div>
);

export default Quote;
