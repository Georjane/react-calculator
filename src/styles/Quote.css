.quote {
  background-color: rgba(231, 228, 216, 0.082);
  height: 80vh;
  padding: 15px;
}

h2 {
  font-weight: lighter;
}
