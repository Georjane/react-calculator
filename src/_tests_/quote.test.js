import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders quotes page', () => {
  const tree = renderer.create(<Quote />);
  expect(tree).toMatchSnapshot();
});
