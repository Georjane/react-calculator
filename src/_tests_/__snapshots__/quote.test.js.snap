// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders quotes page 1`] = `
<div
  className="quote"
>
  <h2>
    <blockquote>
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
    </blockquote>
  </h2>
</div>
`;
